src/lib.rs

```diff
@@ -290,6 +290,23 @@
 //! # fn main() {}
 //! ```
 //!
+//! ## Environment Variable Fallback
+//!
+//! It is possible to specify an environment variable fallback option for arguments, so that their value is taken
+//! from the specified environment variables if not given through the command-line:
+//!
+//! ```
+//! # use structopt::StructOpt;
+//!
+//! #[derive(StructOpt)]
+//! #[structopt(name = "foo")]
+//! struct Foo {
+//!   #[structopt(short = "param", env = "PARAMETER_VALUE")]
```

Comment:
The short form of an argument should generally be a single letter, so maybe `short = "p"`.  Or just use automatic naming like so:

```
 //!   #[structopt(long, short, env = "PARAMETER_VALUE")] 
```

Reply:
Sorry - my bad. Fixed 👍